management/server/grpcserver.go

```diff
@@ -109,17 +111,45 @@ func (s *GRPCServer) GetServerKey(ctx context.Context, req *proto.Empty) (*proto
 	}, nil
 }
 
+// getRealIP returns real peer IP from load balance or fallback to IP of the GRPC peer
```

Comment:
I think there should also be a config option for how many trusted proxies are in front of our server.
With the current behavior, if it's 0, then `X-Forwarded-For` is either spoofed or wrong (added by the client proxy).
If it's n>1, then we would get the wrong IP address. It should be shifted n times to the left

Reply:
Very good point. I think I'll use [the real IP middleware](https://github.com/grpc-ecosystem/go-grpc-middleware/blob/main/interceptors/realip/realip.go) instead of my implementation. See the draft, I only need to make the trusted proxy list configurable.